HousingSearchApi.Tests/V2/E2ETests/Fixtures/fixture_generator/assets.py

```diff
@@ -4,14 +4,16 @@
 from faker import Faker
 
 
-def generate_assets() -> list[dict]:
+def generate_assets(count: int = 2000) -> list[dict]:
```

Comment:
Allow specifying the count - was interesting to test with 100k records for better performance checks

Not a significant change